plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -259,9 +259,11 @@ private static Map<String, Object> serializeStructType(ColumnMappingMode columnM
         fields.put("type", "struct");
         fields.put("fields", rowType.getFields().stream()
                 .map(field -> {
+                    String fieldName = field.getName()
```

Comment:
The commit message is too long. Please shorten that. 
https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Also, have you already sent CLA? 
https://github.com/trinodb/cla

Reply:
Yes, I have already sent the CLA.